### PR TITLE
Package upgrades

### PR DESCRIPTION
**Pull request by lb-**
This pull request upgrades the chalk and commander packages used, adds a note about getting started with development on the readme and adds a few links/details to the package.json file.

### CODE REVIEW

**Comment by lb-** [comment]
gonna close this one as it looks like this will not be merged in - let me know if you want me to re-open.

thanks

**Comment by joshwcomeau** [comment]
Ah! I'm sorry @lb-, I don't believe I ever saw your two pull requests. Truthfully I wasn't aware anyone else really used this package!

Sorry I missed your contributions, they're really appreciated!

File: README.md [hunk 1]
```diff
@@ -162,3 +162,11 @@ This is a brand new thing! I'd like to add more functionality:
...
 - Better error messaging, more edge-case support
 - Editor integrations :o
+
+## Development
+
+To get started with development:
+* Check out this git repo locally, you will need to ensure you have Yarn installed globally.
+* In the folder run `yarn install`
+* Check that command runs `node ../new-component/src/index.js --help`
+* Alternatively you can set up a symlink override by running `npm link` then `new-component --help`. Note: this will override any globally installed version of this package.
```
Comment (joshwcomeau) [inline README.md]: Good tip!

File: package.json [hunk 2]
```diff
@@ -1,7 +1,15 @@
 {
   "name": "new-component",
-  "version": "0.0.1",
+  "version": "1.0.1",
```
Comment (joshwcomeau) [inline package.json]: Huh, not sure how I published `1.0.1` without updating package.json! Good catch

**Pull request merged**